HospitalToday/Logic/IService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using HospitalToday.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HospitalToday.Logic
+{
+    interface IService<T> where T:class
```

Comment:
Add white-spaces after T and before class
`where T : class`

Reply:
Fixed